API tests on ip ranges (#10703)
DataDog/documentation: content/en/synthetics/guide/identify_synthetics_bots.md

**Note:** IPs listed use the CIDR notation and may require conversion to IPv4 address ranges before use.

> **Note:** You can create an API test on the above endpoint with a JSONPath assertion like `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length` to be alerted in case of modification.

Author: ```suggestion
**Note:** Apart from addition of new managed locations IPs, the listed IPs rarely change. That being said, if you want to be alerted in case of modification, you can create an API test on the above endpoint with a JSONPath assertion like `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.
```

> **Note:** Apart from addition of new managed locations IPs, the listed IPs rarely change. That being said, if you want to be alerted in case of modification, you can create an API test on the above endpoint with a JSONPath assertion like `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.

Author: ```suggestion
**Note:** Apart in case of addition of new managed locations IPs, the listed IPs rarely change. That being said, if you want to be alerted in case of modification, you can create an API test on the above endpoint with a JSONPath assertion like `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.
```

> **Note:** Apart in case of addition of new managed locations IPs, the listed IPs rarely change. That being said, if you want to be alerted in case of modification, you can create an API test on the above endpoint with a JSONPath assertion like `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.

Reviewer: ```suggestion
**Note:**Except in the case of new managed locations IPs, the listed IPs rarely change. That being said, if you want to be alerted when the listed IPs change, you can create an API test on the above endpoint with a JSONPath assertion like `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.
```
